tests/test_wave_01.py

```diff
@@ -141,12 +139,10 @@ def test_moves_movie_from_watchlist_to_watched():
     # Assert
     assert len(updated_data["watchlist"]) is 1
     assert len(updated_data["watched"]) is 2
+    assert updated_data["watched"][1]["title"] == "It Came from the Stack Trace"
```

Comment:
👍 We can also use the variables provided in the arrange step:

```suggestion
    assert updated_data["watched"][1] == HORROR_1
```